format/avro/decoders/array.go

```diff
@@ -0,0 +1,51 @@
+package decoders
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/wader/fq/format/avro/schema"
+	"github.com/wader/fq/pkg/decode"
+)
+
+func decodeArrayFn(schema schema.SimplifiedSchema) (func(string, *decode.D), error) {
+	if schema.Items == nil {
+		return nil, errors.New("array schema must have items")
+	}
+
+	valueD, err := DecodeFnForSchema(*schema.Items)
+	if err != nil {
+		return nil, fmt.Errorf("failed getting decode fn for array item: %w", err)
+	}
+
+	//Arrays are encoded as a series of blocks. Each block consists of a long count value, followed by that many array
```

Comment:
I think all comments in fq uses "//<space>..." (unless it's go generate etc), would be nice to keep consistent